chiselFrontend/src/main/scala/chisel3/core/Aggregate.scala

```diff
@@ -10,11 +10,12 @@ import chisel3.internal._
 import chisel3.internal.Builder.pushCommand
 import chisel3.internal.firrtl._
 import chisel3.internal.sourceinfo.{SourceInfo, DeprecatedSourceInfo, VecTransform, SourceInfoTransform}
+import chisel3.NotStrict.CompileOptions
```

Comment:
Correct CompileOptions threading so we don't need an explicit import in chisel3 code (with the possible exception of utils).